### PR TITLE
logging added to model

### PR DESCRIPTION
**Pull request by nirumand**
Since we do not know what happens for success scenario. i added the debugging.

### CODE REVIEW

File: src/main/java/com/nirumand/messaging/rabbitmq/model/Event.java [hunk 1]
```diff
@@ -48,6 +48,7 @@ public Event setMessage(String message) {
 	public String toString() {
 		ObjectMapper objectMapper = new ObjectMapper();
 		try {
+			log.debug("To String method is called");
```
Comment (danielnir) [inline src/main/java/com/nirumand/messaging/rabbitmq/model/Event.java]: I think you should use log.info

File: src/main/java/com/nirumand/messaging/rabbitmq/model/Event.java [hunk 2]
```diff
@@ -48,6 +48,7 @@ public Event setMessage(String message) {
...
 			return objectMapper.writeValueAsString(this);
 		} catch (JsonProcessingException e) {
 			log.error("Can not convert to String",e);
```
Comment (danielnir) [inline src/main/java/com/nirumand/messaging/rabbitmq/model/Event.java]: use log.catching

**Review by danielnir — CHANGES_REQUESTED** [review]
There are some issues

File: src/main/java/com/nirumand/messaging/rabbitmq/model/Event.java (hunk 2 above)
Comment (nirumand) [reply src/main/java/com/nirumand/messaging/rabbitmq/model/Event.java]: log.catching is used with log4j2

File: src/main/java/com/nirumand/messaging/rabbitmq/model/Event.java (hunk 1 above)
Comment (nirumand) [reply src/main/java/com/nirumand/messaging/rabbitmq/model/Event.java]: applied

**Review by danielnir — APPROVED** [review]

**Pull request merged**